<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePage</class>
 <widget class="QWidget" name="ProfilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="containerUsername">
     <item>
      <widget class="QLabel" name="profilePicture">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Profile Picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUsername">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerUsernameTracks">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="parentContainerTracks">
     <item>
      <widget class="QLabel" name="labelTracks">
       <property name="text">
        <string>Piste la plus écoutée</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="containerTracks"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerTracksArtists">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="parentContainerArtists">
     <item>
      <widget class="QLabel" name="labelArtists">
       <property name="text">
        <string>Artiste le plus écouté</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="containerArtists"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerArtistsAlbums">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="parentContainerAlbums">
     <item>
      <widget class="QLabel" name="labelAlbums">
       <property name="text">
        <string>Album le plus écouté</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="containerAlbums"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerAlbumsButtons">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="containerButtons">
     <item>
      <widget class="QPushButton" name="trendingButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Tendances</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recommendationsButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Recommandations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
